app/views/pregnancies/_new_pregnancy.html.erb

```diff
@@ -22,7 +22,7 @@
   </div>
 
   <div class="col-md-2">
-    <%= f.date_field :initial_call_date, label: "Initial Call Date", value: @today %>
+    <%= f.date_field :initial_call_date, label: "Initial Call Date", value: @today,  required: "required" %>
```

Comment:
@colinxfleming ya looking at it more closely I should have made it `required: true` as that's a little more readable. Didn't think of that last night but in the future that's an option.